HomeWork/src/main/java/HomeWork5/HomeWork5.java

```diff
@@ -58,7 +57,7 @@ public static String stringReverse (String reverse) {
         return new  StringBuilder(reverse).reverse().toString();
```

Comment:
1. Please add null validation for an input string
2. Would be better to rename `reverse` to `source` or any other name that suits here better